### PR TITLE
pack: add documentation for '--filename'

### PR DESCRIPTION
**Pull request by better0fdead**
Added documentation for flag '--filename' (command pack).

Part of #703

### CODE REVIEW

File: doc/commands/pack.rst [hunk 1]
```diff
@@ -73,6 +73,10 @@ be it RPM, DEB, TGZ, or a Docker image.
             -   The suffix of the resulting file or image name.
                 For example, a ``tar.gz`` distribution is named according to the pattern:
                 ``<name>-<version>[.<suffix>].<arch>.tar.gz``.
+        *   -   ``--filename``
+            -   Explicitly set a full name of the bundle.
+                For example, a bundle packed with ``--filename bundle_name.tar.gz`` is named:
```
Comment (DifferentialOrange) [inline doc/commands/pack.rst:78]:
```suggestion
                For example, a bundle packed with ``--filename bundle_name.tar.gz`` is named
```

File: doc/commands/pack.rst [hunk 2]
```diff
@@ -73,6 +73,10 @@ be it RPM, DEB, TGZ, or a Docker image.
             -   The suffix of the resulting file or image name.
```
Comment (DifferentialOrange) [inline doc/commands/pack.rst:73]: Add a translation here https://github.com/tarantool/cartridge-cli/blob/master/doc/locale/ru/LC_MESSAGES/doc/commands/pack.po

File: doc/commands/pack.rst (hunk 1 above)
Comment (better0fdead) [reply doc/commands/pack.rst]: Fixed

File: doc/commands/pack.rst (hunk 2 above)
Comment (better0fdead) [reply doc/commands/pack.rst:73]: Added

**Review by LeonidVas — APPROVED** [review]
Hi! Thank you for the patch.
`Part of` -> `Follows up` because the issue was closed.
As for me `documentation` is more appropriate than `pack`. Up to you.

**Pull request merged**